notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Worth adding an explanation about why this schedule is important and why using cosine scheduling? Ideally the link to a paper that introduced the idea.</p><p>(DDPM was using linear and quadratic schedule. And DDIM introduced the cosine scheduling to my understanding.)</p><p>https://arxiv.org/pdf/2102.09672.pdf</p><p> </p><p>"locally" -&gt; "directly"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done.</p>